AppCenterData/AppCenterData/Internal/LocalStore/MSDocumentStore.h

```diff
@@ -87,6 +88,56 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (MSDocumentWrapper *)readWithToken:(MSTokenResult *)token documentId:(NSString *)documentId documentType:(Class)documentType;
 
+/**
+ * List all the documents from the store and return the list if not expired.
+ *
+ * @param token CosmosDB token.
+ * @param partition The CosmosDB partition key.
+ * @param documentType The document type to list.
+ * @param baseOptions Options for listing and storing the documents.
+ *
+ * @return A MSPaginatedDocuments object. List of documents found in the local store for a particular partition.
```

Comment:
```suggestion
 * @return List of documents found in the local store for a particular partition.
```
I guess there's no reason to specify the return type in the comment